resources/assets/uad_lists.json

```diff
@@ -14768,6 +14768,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.uscc.ecid",
+    "list": "Misc",
+    "description": "Probably Call Guardian (anti-spam), or a caller ID service provided by U.S. Cellular.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
```

Comment:
Oh by the way, are you sure it's `expert`?

Reply:
I'd say so since almost everyone will want this. I don't know about any free spam filters/callerids.

Reply:
It isn't a privacy issue, I think, because the carrier already knows who is calling you.

Reply:
My comment was purely based on https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen. What do you think?